### ISSUE TITLE
Filename too long error when trying to add module to our git repo

### ISSUE THREAD

**Issue by TrevorPT** [gruntjs/grunt-contrib-cssmin#266]
Thanks for the handy grunt plugin.  However, when I try to add the module to our repo (so the codebase is build-ready without having to npm any modules), I'm getting a "Filename too long" error on the maxmin module.  Anyone else encounter this?

`warning: unable to access 'public_html/node_modules/grunt-contrib-cssmin/node_modules/maxmin/node_modules/pretty-bytes/node_modules/meow/node_modules/normalize-package-data/node_modules/validate-npm-package-license/node_modules/spdx-correct/node_modules/spdx-license-ids/.gitignore': Filename too long`

**Comment by PaulineNgugi2010** [comment]
Try the Long Path Tool

**Comment by gs-smuthyam** [comment]
This could be a solution for removing an existing file. But how to fix this when the problem is with pull from a git repo?

**Comment by TrevorPT** [comment]
I decided not to add cssmin to our git repo, avoiding the issue, as it's a build tool and not a code dependency.  A colleague suggested adding `longpaths = true` to the git config, but I haven't tried as I no longer need it.  see http://stackoverflow.com/a/34415249 for global and local settings.

**Comment by gs-smuthyam** [comment]
Ok, this is how I fixed this issue for our environment. We want the node modules to be within the repo(ya this is a debatable concept but still).

So What I did is removed the maxmin from within the node_modules of grunt-contrib-cssmin. Checked in the maxmin at the parent level node_modules. 
Since the node scans through the node_modules, the dependency is taken care. 

Interestingly the new maxmin package(2.1.x) doesn't have too many dependencies it seems.  May be you want to change the dependency from 1.X to 2.X and try it out.
